### DISCUSSION TITLE
Do NOT use your IP to host the bot !

### DISCUSSION THREAD

**Discussion by hermitoff** [jagrosh/MusicBot#744] [General]
Hello,

Do not use your IP to host the bot!
Because anyone can find it.

You just have to send to the bot a request (with the command play) to an ip logger to get it.

Here is an example to show you:
![unknown](https://user-images.githubusercontent.com/82811865/131243025-01dec083-03d0-4876-b9ba-0669b103bf59.png)

**Comment by MichailiK** [comment]
Disallowing external URL sources & preventing such IP leaks is now tracked in issue #808
